src/assets/styles/base.scss

```diff
@@ -68,6 +68,10 @@ p {
   font-weight: 400;
 }
 
+.app-action {
+  color: $app-blue !important;
```

Comment:
Do we really need !important? If not, we'd better not use it.

Reply:
IMHO, if we use a base class, we hope it overrides any styles set in the component itself, so I don't mind the "important".

Reply:
ok, yeah, no problem in this case. I just prefer not seeing any of them unless in rare cases but as we have already in many places in the project it's ok. These !importants may cause tricky issues if we start using them without discernment.

Reply:
We might be able to get away with NOT using it, but you run the risk of Vuetify overriding depending on where it is used.

Reply:
If we don't include "important", and we do want the style to override something else, what's the alternative? Another class with important, eg, "app-action-i" ? Yuck.
(Yes, CSS uses the most specific selector, but that's not something we should have in the base style file.)

Reply:
> These !importants may cause tricky issues if we start using them without discernment.

Absolutely. My approach is styles in the component when they are specific, and styles in global assets otherwise. This seems to work OK mostly.